### PR TITLE
add routing to home, about, help

### LINKED ISSUE #41: Add routing for about page, etc.

### PR DESCRIPTION
**Pull request by charkour**
This adds basic routing to the home page (the main tool), and about page, and a help page. 

There is no styling on the navigation yet. Do you guys have any ideas on how I should look. Maybe styling can be another PR if someone is willing to figure out a good system for navigating. 

Test by running `npm i` and `npm start` and then you can click the links on the page to route to different pages. 

closes #41 

Update
-----

- Made the routing cleaner by putting links on the nav bar
- Made the import/export buttons more consistent looking and behaving
- Split up large files into smaller ones

Please let me know what you like and don't like! I just went with my first instinct. Sorry, there are a fair number of changes, so I'll try to keep things smaller in the future.

### CODE REVIEW

File: client-course-schedulizer/src/components/App/App.tsx [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
+          <div>
+            <Link to="/">Home</Link>
+          </div>
+          <div>
+            <Link to="/about">About</Link>
+          </div>
+          <div>
+            <Link to="/help">Help</Link>
+          </div>
+        </nav>
```
Comment (Jdellis818) [inline client-course-schedulizer/src/components/App/App.tsx]: Is it possible to put these links inside the hamburger menu?

**Review by Jdellis818 — CHANGES_REQUESTED** [review]
This looks like a good start to me. No need to put too much effort into styling, but I think I'd just prefer to see the links inside the hamburger menu popover if that's possible.

**Comment by charkour** [comment]
Okay, I think that works, but it's a little confusing having different actions in one menu. I'll try it out.

**Review by Jdellis818 — APPROVED** [review]
This is awesome! Really well done. It looks pretty slick

**Pull request merged**